src/state/users/actions.js

```diff
@@ -16,13 +16,38 @@ export const setUsers = users => ({
   },
 });
 
+export const setSelectedUserDetail = user => ({
+  type: SELECTED_USER,
```

Comment:
Please rename in SET_SELECTED_USER, because this is an action but there is no verb.
We could have also CLEAR_SELECTED_USER, REMOVE_SELECTED_USER, UPDATE_SELECTED_USER and so on, so it's better to have the verb specified